pkg/section/section.go

```diff
@@ -28,6 +28,7 @@ func (list SectionList) String() []string {
 }
 
 func DefaultSections() SectionList {
+	// pointer for Module since we need to mutate it
```

Comment:
To Module?

Reply:
> To Module?

Sorry, my first attempt was a bit of hacking things together, so some of the comments were a bit rough :slightly_smiling_face: